### PR TITLE
doc for BurntPancakeGraph

### PR DESCRIPTION
**Pull request by marinaanagno**

### CODE REVIEW

File: doc/examples.xml [hunk 1]
```diff
@@ -533,6 +533,43 @@ true]]></Example>
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="BurntPancakeGraph">
+<ManSection>
+  <Oper Name="BurntPancakeGraph" Arg="[filt, ]n"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    If <A>n</A> is a positive integer, then this operation returns
+    the <E>Burnt Pancake graph</E> with <M>n!</M> vertices and <M>n!(n - 1)</M>
```
Comment (james-d-mitchell) [inline doc/examples.xml]: Burnt Pancake -> burnt pancake

File: doc/examples.xml [hunk 2]
```diff
@@ -533,6 +533,43 @@ true]]></Example>
...
+    directed edges. The <M>n</M>th Pancake graph is the Cayley graph of the
```
Comment (james-d-mitchell) [inline doc/examples.xml]: Pancake -> pancake

File: doc/examples.xml [hunk 3]
```diff
@@ -533,6 +533,43 @@ true]]></Example>
...
+    hyperoctahedral group acting on <C>[<A>-n</A> .. -1, 1 .. <A>n</A>]</C> with respect to the
+    generating set consisting of the <Q>prefix reversals</Q>, which are defined in exactly
+    the same way as in  <Ref Attr="PancakeGraph"/>. The hyperoctahedral group consists of
```
Comment (james-d-mitchell) [inline doc/examples.xml]: Isn't `PancakeGraph` an operation? If so, this should be `<Ref Oper=`.

File: doc/examples.xml [hunk 4]
```diff
@@ -533,6 +533,43 @@ true]]></Example>
...
+    permutations <M>p</M> acting on <C>[<A>-n</A> .. -1, 1 .. <A>n</A>]</C>, where the image
+    of any arbitrary point <M>i</M> in <C>[<A>-n</A> .. -1, 1 .. <A>n</A>]</C> is equal to the
```
Comment (james-d-mitchell) [inline doc/examples.xml]: "any arbitrary" -> "every".

File: doc/examples.xml (hunk 3 above)
Comment (marinaanagno) [reply doc/examples.xml]: fixing that :)

File: doc/examples.xml [hunk 5]
```diff
@@ -533,6 +533,43 @@ true]]></Example>
...
+    negative of the image of <M>-i</M> under <M>p</M>.
```
Comment (james-d-mitchell) [inline doc/examples.xml:549]: It'd be good to mention that in GAP permutations act on the positive integers only, and as such the permutations of -n to n (without 0) are represented by ???

Comment (marinaanagno) [reply doc/examples.xml:549]: I don't know how I didn't push this, sorry, I wrote it and then deleted it by mistake probably. Fixing this now :)

**Pull request merged**